### ISSUE TITLE
lein repl span a new jvm process every time.

### ISSUE THREAD

**Issue by wuhx** [ninjudd/drip#61]
drip works alright if invoke from  command line directly . but after add LEIN_JAVA_CMD=${LEIN_JAVA_CMD-drip} to leinrc, lein repl didn't reuse exist jvm process.

software version:
mac os 10.8.2 
lein --version
Leiningen 2.0.0 on Java 1.7.0_05 Java HotSpot(TM) 64-Bit Server VM
drip 0.2.3 . install through brew.

after run lein repl three times.

`--> drip ps
10917 org.flatland.drip.Main clojure.main /Users/wuhx/.drip/0.2.3/2710892dbd2e359c5c9a5a1c23d9101fb97e0671/10891-1 -XX:+TieredCompilation -Xbootclasspath/a:/Users/wuhx/.lein/self-installs/leiningen-2.0.0-standalone.jar -Dfile.encoding=UTF-8 -Dmaven.wagon.http.ssl.easy=false -Dleiningen.original.pwd=/Users/wuhx/Desktop/work/mybuy -Dleiningen.script=/Users/wuhx/bin/lein -Djava.awt.headless=true
10841 org.flatland.drip.Main clojure.main /Users/wuhx/.drip/0.2.3/2710892dbd2e359c5c9a5a1c23d9101fb97e0671/10817-1 -XX:+TieredCompilation -Xbootclasspath/a:/Users/wuhx/.lein/self-installs/leiningen-2.0.0-standalone.jar -Dfile.encoding=UTF-8 -Dmaven.wagon.http.ssl.easy=false -Dleiningen.original.pwd=/Users/wuhx/Desktop/work/mybuy -Dleiningen.script=/Users/wuhx/bin/lein -Djava.awt.headless=true
11137 org.flatland.drip.Main clojure.main /Users/wuhx/.drip/0.2.3/2710892dbd2e359c5c9a5a1c23d9101fb97e0671/11110-1 -XX:+TieredCompilation -Xbootclasspath/a:/Users/wuhx/.lein/self-installs/leiningen-2.0.0-standalone.jar -Dfile.encoding=UTF-8 -Dmaven.wagon.http.ssl.easy=false -Dleiningen.original.pwd=/Users/wuhx/Desktop/work/mybuy -Dleiningen.script=/Users/wuhx/bin/lein -Djava.awt.headless=true

**Comment by arichiardi** [comment]
I confirm this, I have just tried and ```drip ps``` returns a JVM per ```lein``` command.

**Comment by arichiardi** [comment]
Hello guys, any update on this?

**Comment by Raynes** [comment]
Besides merging pull requests from time to time, my understanding is that this project is more or less abandoned. Anyone is welcome to take it on.

**Comment by ninjudd** [comment]
Yeah. I made @headius a contributer a while back, but I suspect he's even more busy than I am. I'm happy to make anyone who wants to help out a contributer.
